packages/website/docusaurusConfig.ts

```diff
@@ -66,7 +66,7 @@ const themeConfig: ThemeCommonConfig & AlgoliaThemeConfig = {
   ],
   image: 'img/logo-twitter-card.png',
   navbar: {
-    title: 'TypeScript ESLint',
+    title: 'typescript-eslint',
```

Comment:
@JamesHenry @Josh-Cena this is why I'd gone towards solidifying as _TypeScript-ESLint_ at first. IMO it looks a little odd to have the project name be lower-kebab-case on the homepage:

<img width="1470" alt="Screenshot of the deploy preview homepage with lower-kebab-case 'typescript-eslint' as the title" src="https://user-images.githubusercontent.com/3335181/205434978-2f013fd4-4b75-41f1-9e1d-be73093e6dbe.png">

My logic was:
1. I think it's good to keep the title casing & general phrasing consistent where possible
1. For a project homepage, it makes sense to keep things in title case
1. The hyphen is useful for showing it's a single project
1. Therefore, _TypeScript-ESLint_ is the logical conclusion

Comment reference to counterpoints: https://github.com/typescript-eslint/typescript-eslint/pull/6147#pullrequestreview-1202034772

Thoughts?

Reply:
LGTM!

Reply:
We can make the hero prettier in general in a follow up as we mentioned

Reply:
webpack and many others have the same approach, but they mostly combine the word with the logo, we can do the same

Reply:
Yeah I've started playing with this locally and _really_ like putting that logo there. I'll throw up a few examples soon 😄